build.cake

```diff
@@ -227,7 +251,7 @@ Task("Publish-MyGet")
         throw new InvalidOperationException("Could not resolve MyGet API key.");
     }
 
-    foreach(var package in new[] { "Cake", "Cake.Core", "Cake.Common", "Cake.Testing" })
+    foreach(var package in new[] { "Cake", "Cake.Core", "Cake.Common", "Cake.Testing", "cake.portable" })
```

Comment:
As discussed, going to leave this the way it is, based on recommendations from Chocolatey:

https://github.com/chocolatey/chocolatey/wiki/CreatePackages#naming-your-package